e2e/pod.go

```diff
@@ -542,3 +553,73 @@ func validateRWOPPodCreation(
 
 	return nil
 }
+
+// verifySeLinuxMountOption verifies the SeLinux context MountOption added to PV.Spec.MountOption
+// is successfully used by nodeplugin during mounting by checking for its presence in the
+// nodeplugin container logs.
+func verifySeLinuxMountOption(
+	f *framework.Framework,
+	pvcPath, appPath, daemonSetName, cn, ns string,
+) error {
+	mountOption := "context=\"system_u:object_r:container_file_t:s0:c0,c1\""
+
+	// create PVC
+	pvc, err := loadPVC(pvcPath)
+	if err != nil {
+		framework.Failf("failed to load pvc: %v", err)
```

Comment:
It would be best to return an error from this point. We shouldn't let the internal function fail due to this error. Instead, we should return an error in all cases and let the caller function handle the failure.

Reply:
> It would be best to return an error from this point. We shouldn't let the internal function fail due to this error. Instead, we should return an error in all cases and let the caller function handle the failure.

Thanks !
missed it while abstracting the calls to a helper function.
its fixed now.